src/main/java/befaster/runner/ClientRunner.java

```diff
@@ -42,13 +47,31 @@ public ClientRunner withSolutionFor(String methodName, UserImplementation soluti
         return this;
     }
 
+    public ClientRunner withJourneyId(String journeyId) {
+        this.journeyId = journeyId;
+        return this;
+    }
+
 
     public void start(String[] args) {
         if(!isRecordingSystemOk()) {
             System.out.println("Please run `record_screen_and_upload` before continuing.");
             return;
         }
 
+        ChallengeServerClient challengeServerClient = new ChallengeServerClient(hostname, journeyId, true);
+
+        try {
+            System.out.println(challengeServerClient.get(JOURNEY_PROGRESS_ENDPOINT));
```

Comment:
I am having second thoughts about the inlining, `challengeServerClient.getJourneyProgress()` looked better.